components/serverless/internal/controllers/kubernetes/configmap_service.go

```diff
@@ -14,10 +15,13 @@ import (
 	"github.com/kyma-project/serverless/components/serverless/internal/resource"
 )
 
+const cfgConfigMapFinalizerName = "serverless.kyma-project.io/config-map-finalizer"
```

Comment:
We know that this finalizer is for the config map. I propose a more general name.
```suggestion
const cfgConfigMapFinalizerName = "serverless.kyma-project.io/deletion-hook"
```